<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WordCounterClass</class>
 <widget class="QMainWindow" name="WordCounterClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Word Counter</string>
  </property>
  <widget class="QWidget" name="m_centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTableView" name="m_tableView">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="m_menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>319</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="m_menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="m_actionOpen"/>
    <addaction name="separator"/>
    <addaction name="m_actionExit"/>
   </widget>
   <addaction name="m_menuFile"/>
  </widget>
  <action name="m_actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="m_actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="WordCounter.qrc"/>
 </resources>
 <connections/>
</ui>
